.travis.yml

```diff
@@ -1,16 +1,15 @@
-language: go
 sudo: false
+language: go
 
 go:
+  - 1.6.x
   - 1.7.x
 
 services:
   - redis-server
   - mongodb
 
 install:
-  - go get -d -v ./...
-  - go get -d -v -tags coprocess ./...
```

Comment:
See discussion on slack. They do nothing, as they would only download but everything is already on disk since all deps are vendored.